### DISCUSSION TITLE
Synchronization of Add/Update user

### DISCUSSION THREAD

**Discussion by YoumnaJ** [OrchardCMS/OrchardCore#15164] [Q&A]
Hi , I need to know if there is a solution to synchronize add/update users between orchard core solution and another solution using another CMS for example when I add or update a user in the solution not using orchard CMS I need this user to be updated or added in my orchard core solution. Can I find a default api or something like api can help me to do this.
thank you

**Comment by MichaelPetrinolis** [comment]
What you can do now without custom coding is to configure OpenId Client and the Users Registration features to allow users from an OpenId server to authenticate. Then you can update the user's registration script (or write custom code in a module), to assign application roles to the external users based on the provided claims.

In your case, you have two options.
i. Configure the other CMS to be an OpenId server (this can be easily done with OrchardCore, thanks to OpenId Server feature which is based on OpenIdDict )
ii. Use an external OpenId server and configure both CMS to authenticate users from it. You can also use Azure EntraId which is also provided in OC and has a nice [guide](https://docs.orchardcore.net/en/latest/docs/guides/microsoft-entra-id-integration/) in the documentation.
